frontend/cypress/integration/editview/filetree.spec.js

```diff
@@ -0,0 +1,36 @@
+describe('When opening repository for editing', () => {
+  beforeEach(() => {
+    cy.deleteUser('testuser')
```

Comment:
deleteUser is now deprecated, and replaced with resetUsers and resetToken functions committed by @jkotima in token-service-integration branch (#79), later merged into staging. 
```suggestion
    cy.resetUsers()
    cy.resetTokens()
```